docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,274 @@
+In this tutorial, we prepare a PyTorch Fashion-MNIST example which completed by BigDL and supports running on yarn cluster. In particular, we will show you:
+* How to prepare for the environment before submitting the program to Yarn;
+* How to execute the BigDL program on Yarn through multiple ways;
+
+BigDL Orca example code can be found here [Fashion_MNIST Example](https://github.com/intel-analytics/BigDL/tree/main/docs/docs/tutorials/tutorial_example/Fashion_MNIST/).
+
+# Key Concepts
+## OrcaContext
```

Comment:
1. Change to `init_orca_context`
2. Make it clear that every Orca program should call `init_orca_context` at the beginning 
3. Describe what `init_orca_context` does, and what its API looks like